packages/_private/vue-demo/src/DemoForm.vue

```diff
@@ -0,0 +1,339 @@
+}
+
+const submit = (e: Event) => {
+  e.preventDefault();
+  e.stopPropagation();
+
+  const formElement = e.target as HTMLFormElement;
+  const isValid = formElement.checkValidity();
+
+  if (isValid) {
+    // eslint-disable-next-line no-alert
+    alert('Your data was successfully submitted');
+  }
+}
+
+const synChange = (e: SynChangeEvent) => {
+  const form = formRef.value;
+
+  const normalizedData = normalizeData(new FormData(form));
+
+  // Log the normalized data
+  console.log(normalizedData);
+
+  // Set the field into state
+  const element = e.target as FormEnabledElements;
+  const { checked, name, value } = element;
+
+  const finalValue = typeof checked !== 'undefined'
+    ? !!checked
+    : value;
+
+  formData.value = {
+    ...formData.value,
+    [name]: finalValue,
+  }
+};
+</script>
+
+<template>
+  <form
+    @reset="reset"
+    @submit="submit"
+    @syn-change="synChange"
+    ref="formRef"
+  >
+    <!-- PersonalInformation -->
+    <DemoFieldset legend="Personal Information">
+
+      <SynVueRadioGroup
+        id="radiogroup-gender"
+        name="gender"
+        label="Please tell us your gender"
+        required
+        v-model="formData.gender"
+      >
+        <SynVueRadio value="f">Female</SynVueRadio>
+        <SynVueRadio value="m">Male</SynVueRadio>
+        <SynVueRadio value="other">Other</SynVueRadio>
+      </SynVueRadioGroup>
+
+      <SynVueSelect
+        id="select-role"
+        label="Current position"
+        name="role"
+        required
+        v-model="formData.role"
+      >
+        <SynVueOptgroup label="Developers">
+          <SynVueOption value="backend">Backend Developer</SynVueOption>
+          <SynVueOption value="frontend">Frontend Developer</SynVueOption>
+        </SynVueOptgroup>
+        <SynVueOptgroup label="Other">
+          <SynVueOption value="lead">Team Lead</SynVueOption>
+          <SynVueOption value="other">Other (please specify in comment section below)</SynVueOption>
+        </SynVueOptgroup>
+      </SynVueSelect>
+
+      <SynVueInput
+        id="input-text"
+        label="Name"
+        :minlength="5"
+        :maxlength="20"
+        name="name"
+        placeholder="Please insert a value for the regular text input (between 5 and 20 Characters)"
+        required
+        v-model="formData.name"
+        type="text"
+      />
+
+      <SynVueInput
+        id="input-email"
+        label="E-Mail"
+        name="email"
+        placeholder="Please insert your E-mail address"
+        required
+        v-model="formData.email"
+        type="email"
+      />
+
+      <SynVueInput
+        id="input-phone"
+        label="Phone"
+        name="phone"
+        placeholder="Please provide your phone number"
+        required
+        v-model="formData.phone"
+        type="tel"
+      />
+
+      <SynVueInput
+        id="input-date"
+        label="Date of birth"
+        name="date"
+        placeholder="Please insert your E-mail address"
```

Comment:
Should be removed. Is the message from email and a placeholder is not needed for date anyway. This needs to be fixed in all wrappers